docs/en/reference/schema-representation.rst

```diff
@@ -93,7 +93,18 @@ and absolutely not portable.
 -  **engine** (string): The DB engine used for the table. Currently only supported on MySQL.
 
 -  **unlogged** (boolean): Set a PostgreSQL table type as
-  `unlogged <https://www.postgresql.org/docs/current/sql-createtable.htmll>`_
+   `unlogged <https://www.postgresql.org/docs/current/sql-createtable.html#SQL-CREATETABLE-UNLOGGED>`_
+
+-  **temporary** (boolean): Set a PostgreSQL table type as
+   `temporary <https://www.postgresql.org/docs/current/sql-createtable.html#SQL-CREATETABLE-TEMPORARY>`_
```

Comment:
This doesn't look like a PostgreSQL-specific concept to me :thinking:

Reply:
Well, it isn't per se but since I have access to PostgreSQL mainly, I implemented it just on the PostgreSQL platform.